spaceapi_server/src/lib.rs

```diff
@@ -87,6 +89,21 @@ impl SpaceapiServer {
         }
     }
 
+    fn update_values(&self, map: &urlencoded::QueryMap) -> IronResult<Response> {
```

Comment:
Maybe we could create a separate trait for this, similar to the `middleware::Handler` below?